raiden/tests/integration/fixtures/raiden_network.py

```diff
@@ -154,6 +157,26 @@ def monitoring_service_contract_address() -> Address:
     return Address(bytes([1] * 20))
 
 
+@pytest.fixture
+def resolvers(resolver_ports) -> List[Optional[subprocess.Popen]]:
```

Comment:
Can you add a docstring to this fixture too? To explain exactly what it does and why it's here?

Reply:
done